### ISSUE TITLE
[Question] Missing tests for word-diff feature

### ISSUE THREAD

**Issue by MisanthropicBit** [rhysd/git-messenger.vim#72]
As mentioned [here](https://github.com/rhysd/git-messenger.vim/pull/69#issuecomment-771844576), I was still in the process of making tests for the word-diff feature #69.

I can still create them but I didn't get that far if you want to create them yourself :)

**Comment by rhysd** [comment]
Yes, adding unit tests is welcome. I think test cases for `d -> d` and `d -> D` are helpful to be referred.

**Comment by MisanthropicBit** [comment]
> I think test cases for d -> d and d -> D are helpful for understanding how to test them.

Thanks, I based my tests on those.

I can certainly work on the tests although I don't know when they'll be ready so if you prefer to make them yourself then by all means do. If not, I'll get started when I have time :)

**Comment by rhysd** [comment]
OK, I'll leave comment here when I start the work.
